selfservice/strategy/oidc/provider_config.go

```diff
@@ -177,6 +177,7 @@ var supportedProviders = map[string]func(config *Configuration, reg Dependencies
 	"patreon":     NewProviderPatreon,
 	"lark":        NewProviderLark,
 	"x":           NewProviderX,
+	"line_v2_1":   NewProviderLineV21,
```

Comment:
Is the versioning required here? Since we don't support line right now we can probably just call it line :)

```suggestion
	"line":   NewProviderLineV21,
```